src/test/java/zd/zero/waifu/motivator/plugin/assets/RemoteAssetDefinitionServiceTest.kt

```diff
@@ -41,8 +46,9 @@ class RemoteAssetDefinitionServiceTest {
     }
 
     @Test
-    fun getRandomAssetByCategoryShouldReturnLocalWhenNoRemoteAssets() {
-        every { remoteAssetManager.supplyAssetDefinitions() } returns listOf()
+    fun `get random asset by category should return local when no remote assets`() {
+        every { remoteAssetManager.supplyAssetDefinitions() }
```

Comment:
What's up with the mockk set ups?

Reply:
Whoops, I think it got `d$` 😅

Reply:
Whoops, this won't be removed, I'll restore the setups